doc/source/io.rst

```diff
@@ -120,7 +120,8 @@ header : int or list of ints, default ``'infer'``
   rather than the first line of the file.
 names : array-like, default ``None``
   List of column names to use. If file contains no header row, then you should
-  explicitly pass ``header=None``.
+  explicitly pass ``header=None``. If this list contains any duplicates, then
+  you should ensure that ``mangle_dupe_cols=True``.
```

Comment:
what will a user think about this? maybe say that duplicates ``names`` are not allowed unless ``mangle_dups_cols=True`` (which is the default)?